src/OWML.Launcher/OWML.Manifest.json

```diff
@@ -2,7 +2,7 @@
   "author": "Alek",
   "name": "OWML",
   "uniqueName": "Alek.OWML",
-  "version": "0.7.3",
+  "version": "1.0.1",
```

Comment:
i thought this was version 1 🤔

Reply:
a long time ago i created a nuget package with version 1.0.0. it's unlisted but can't be removed :(